test/bench/etc/run_granular_benchmarks.js

```diff
@@ -0,0 +1,93 @@
+'use strict';
+
+/**
+ * This script runs all benchmarks in test/bench/lib/granular and merges their output
+ */
+
+const cp = require('child_process');
+const fs = require('fs/promises');
+const path = require('path');
+const { Task } = require('dbx-js-tools/packages/bson-bench');
+
+const BENCHMARK_REGEX = /(.*)\.bench\.js$/;
+const BENCHMARK_PATH = path.resolve(`${__dirname}/../lib/granular`);
+const DOCUMENT_ROOT = path.resolve(`${__dirname}/../documents`);
+const LIBRARY_PATH = path.resolve(`${__dirname}/../../../.`);
+(async () => {
+  // HACK : run one dummy task with the local bson to ensure it's available for subsequent suites
+  await new Task({
+    documentPath: path.resolve(`${DOCUMENT_ROOT}/binary_small.json`),
+    library: `bson:${LIBRARY_PATH}`,
+    iterations: 1,
+    warmup: 1,
+    operation: 'deserialize',
+    options: {}
+  })
+    .run()
+    .catch(() => null);
+
+  // Run all benchmark files
+  const lib = await fs.readdir(BENCHMARK_PATH);
+  for await (const dirent of lib) {
+    if (BENCHMARK_REGEX.test(dirent)) {
+      const child = cp.fork(`${BENCHMARK_PATH}/${dirent}`);
+      if (child.stdout) child.stdout.pipe(process.stdout);
+      if (child.stderr) child.stdout.pipe(process.stderr);
+
+      await new Promise((resolve, reject) =>
+        child.once('exit', code => {
+          if (code !== 0) return reject();
+          return resolve();
+        })
+      );
```

Comment:
Could we use `import {once} from 'events'`? just trying to simplify manual promise construction.